recipes/r-n1qn1/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set version = '6.0.1-3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-n1qn1
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/n1qn1_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/n1qn1/n1qn1_{{ version }}.tar.gz
+  sha256: f1bb00eeaafe6ac014046a15c095be46be45ecae7e1c84d65099ad2c20e923cc
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - libopenblas                # [unix]
```

Comment:
This is wrong. See https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
Definetly, this was done only to fix an error with some blas library not packaged. Blas should not be a dependency of this package IMO. What do you propose?

Reply:
You can't see original error as this repo was rebased. See similar issue in PR #10298 ([log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=99081&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0) solved by adding `libopenblas` as dependency.

Reply:
> What do you propose?

Change this to `libblas` instead as in the docs link I mentioned.